ldk/goja/src/ui/ui.test.ts

```diff
@@ -0,0 +1,54 @@
+import { UIImpl } from '.';
+import { mocked } from 'ts-jest/utils';
+import any = jasmine.any;
+import anything = jasmine.anything;
```

Comment:
Are these used anywhere? If they are, I think we should be using the [Jest Any Matcher](https://jestjs.io/docs/expect#expectanyconstructor) instead of jasmine.

Reply:
Yeah, I could do that. I was just following the pattern from the clipboard sensor